pkgs/tools/misc/byobu/default.nix

```diff
@@ -11,26 +16,51 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
-  buildInputs = [ python3 perl ];
-  propagatedBuildInputs = [ textual-window-manager ];
+  buildInputs = [ perl pythonEnv makeWrapper ];
+  propagatedBuildInputs = [ textual-window-manager screen pythonEnv ];
 
-  meta = {
+  postUnpack = ''
+    substituteInPlace $sourceRoot/usr/bin/byobu-select-profile.in \
+      --replace "/bin/true" "true" \
+      --replace "gettext" "${gettext}/bin/gettext"
+    substituteInPlace $sourceRoot/usr/bin/byobu-export.in \
+      --replace "gettext" "${gettext}/bin/gettext"
+  '';
+
+  postInstall = ''
+    # Override the symlinks otherwise they fuck with the wrapping
```

Comment:
```suggestion
    # Override the symlinks otherwise they mess with the wrapping
```

Reply:
:laughing: I had some strong feelings about the mess that is byobu at the time I wrote this.